governance-exercises/VoteWizard_poll_exercise_02/Derecognition of Clive AD Due to Compromised Operational Security - July 28, 2023.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Derecognition of Clive AD Due to Compromised Operational Security - JULY 28, 2023
```

Comment:
Caps for date? A choice or unintentional? Assuming you're applying it consistently, works fine.